### ISSUE TITLE
Shift + Alt + F changes are different for typescript

### ISSUE THREAD

**Issue by mimipaskova** [Microsoft/vscode#37141]
I have used this shortcut last few months for the same files and today when I tried to format same files they changed so much ( there are examples) I want same settings as before, how can I change them?

- VSCode Version: 1.17.1
- OS Version: Win 10; version 1703

Steps to Reproduce:

1. Open typescript file (Angular 4). 
2. Click `shift` + `Alt` + `F`
3. Change ' with " everywhere (in imports, in strings). After that my tslint says: " should be ' .
4. Change `(x) =>` to `x => ` in all arrow functions.

``
import { Observable } from 'rxjs/Observable';
import { EventEmitter } from '@angular/core'; ``
to

`` import { Observable } from "rxjs/Observable";
import { EventEmitter } from "@angular/core"; ``

` (res) => {`  to ` res => {`

Reproduces without extensions: Yes

**Comment by mjbvz** [comment]
Are you sure this is not coming from an extension? The formatting changes you describe are not supported by the builtin ts formatter. Please try disabling all extensions to check this

**Comment by mimipaskova** [comment]
![vsc formatting](https://user-images.githubusercontent.com/3625019/32182709-03c98138-bda0-11e7-9c1d-c8d27afc0934.png)

I disabled all installed extensions.

**Comment by mjbvz** [comment]
Did you reload vscode after disabling them? After doing this, you should no longer see tslint warnings or errors in your code

**Comment by mimipaskova** [comment]
After restart everything looks like befoce except deleting nel line in the end of the file. (Maybe this is from any extention).

Thanks

**Comment by mjbvz** [comment]
Yes, please try to identify which extension is causing this issue and report any issues against them

Closing as caused by extension

**Comment by mimipaskova** [comment]
I found the extension - Prettier - JavaScript formatter. It is part of Angular Essentials and it's added in version 0.3.0 (2017-10-18).
 It's impossible to Disable only it and I disabled Angular Essentials and abled all other sub-extensions of Angular Essentials.
If you know how to able Angular Essentials and disable only Prettier I will be happy to hear.  (Or maybe to change settings of Prettier)
